target/linux/imx/image/cortexa7.mk

```diff
@@ -10,3 +12,21 @@ define Device/Default
   KERNEL_LOADADDR := 0x80008000
   IMAGES :=
 endef
+
+define Device/pico-pi-imx7d
```

Comment:
@Leo-PL please make it `vendor_product`. This way you could also drop `SUPPORTED_DEVICES`.

Reply:
Makes total sense - done.